### DISCUSSION TITLE
Infolist inside modal

### DISCUSSION THREAD

**Discussion by clinvest** [filamentphp/filament#9586] [Help] [answered]
Hi all,

Is there a way to have an infolist inside a modal? do I have to make a specific view for that?

There is a form option and form disabled but I'd like to have an infolist. I don't find information in the doc about that.

Thank you in advance,

**Comment by leandrocfe** [comment]
Could you explain more about your question please?
If you want to use an infolist instead of a disabled form, you should do [this](https://filamentphp.com/docs/3.x/panels/resources/viewing-records)

**Comment by coolsam726** [comment, marked as answer]
Assuming you are asking about adding an infolist to an Action, you can simply use `infolist` as follows:
```php
Actions\Action::make('my-action')->label('My Action')->infolist([
      TextEntry::make('name'),
      TextEntry::make('description')
  ])
```

**Reply by clinvest** [reply]
Perfect! thank you very much. Didn't know I could do that with a modal (I didn't find it in the doc).

**Reply by eliezerben** [reply]
> Assuming you are asking about adding an infolist to an Action, you can simply use `infolist` as follows:
> 
> ```
> Actions\Action::make('my-action')->label('My Action')->infolist([
>       TextEntry::make('name'),
>       TextEntry::make('description')
>   ])
> ```

Is there a 'fillForm()' equivalet for infolist when used with an action?
